platform/security/security-migratable/src/test/java/org/codice/ddf/security/migratable/impl/SecurityMigratableTest.java

```diff
@@ -0,0 +1,147 @@
+            "etc/ws-security/server/encryption.properties";
+
+    private static final boolean IS_OPTIONAL = false;
+
+    private Path ddfHome;
+
+    private Path exportDirectory;
+
+    @Before
+    public void setup() throws Exception {
+        ddfHome = Paths.get(DDF_BASE_DIR);
+        exportDirectory = ddfHome.resolve(EXPORTED_REL_PATH);
+    }
+
+    @Test
+    public void testExportValidRelativePaths() throws Exception {
+        // Setup
+        MigratableUtil migratableUtil = mock(MigratableUtil.class);
+        SecurityMigratable securityMigratable = new SecurityMigratable(DESCRIPTION,
+                IS_OPTIONAL,
+                migratableUtil);
+
+        // Perform Test
+        securityMigratable.export(exportDirectory);
+
+        //Verify CRL File was exported
+        assertCrlExport(migratableUtil);
+
+        //verify PDP Directory was exported
+        assertPdpDirectoryExport(migratableUtil);
+    }
+
+    @Test
+    public void testWarningsReturned() throws Exception {
+        //setup
+        MigratableUtil migratableUtil = mock(MigratableUtil.class);
+        MigrationWarning expectedWarning1 = new MigrationWarning("Expected Warning 1");
+        MigrationWarning expectedWarning2 = new MigrationWarning("Expected Warning 2");
+        List<MigrationWarning> expectedWarnings = new ArrayList<>();
+        expectedWarnings.add(expectedWarning1);
+        expectedWarnings.add(expectedWarning2);
+
+        doAnswer(new Answer<Void>() {
+            @SuppressWarnings("unchecked")
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                ((Collection<MigrationWarning>) args[3]).add(expectedWarning1);
+                return null;
+            }
+        })      .when(migratableUtil)
+                .copyFileFromJavaPropertyValue(eq(Paths.get(FILE_CONTAINING_CRL_LOCATION)),
+                        eq(CRL_PROP_KEY),
+                        eq(exportDirectory),
+                        Matchers.<Collection<MigrationWarning>>any());
+
+        doAnswer(new Answer<Void>() {
+            @SuppressWarnings("unchecked")
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                ((Collection<MigrationWarning>) args[2]).add(expectedWarning2);
+                return null;
+            }
+        })      .when(migratableUtil)
+                .copyDirectory(eq(Paths.get(PDP_POLICIES_DIR_REL_PATH)),
+                        eq(exportDirectory),
+                        Matchers.<Collection<MigrationWarning>>any());
+
+        // Perform Test
+        SecurityMigratable securityMigratable = new SecurityMigratable(DESCRIPTION,
+                IS_OPTIONAL,
+                migratableUtil);
+        MigrationMetadata migrationMetadata = securityMigratable.export(exportDirectory);
+
+        // Verify
+        assertThat(migrationMetadata.getMigrationWarnings()
```

Comment:
I _think_ that `containsInAnyOrder` already ensures that the list has as many elements as the array provided so this check _may_ be redundant.